ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-ref.yaml

```diff
@@ -15,6 +15,14 @@ ref:
     default: "false"
     documentation: |-
       Detemine if OTA test will be included
+  - name: CAPABILITIES_IN_CREDENTIALREQUEST
+    default: ""
+    documentation: |-
+      This is a variable uesd for OTA test. It has different values in different versions. It represents the capabilities with capability annotation in CredentialsRequest.
+  - name: ADDITIONAL_ENABLED_CAPABILITIES
```

Comment:
In postupgrade step, seem like this var is not needed.

Reply:
removed